app/assets/stylesheets/layout/page_specific/_content_hub.scss

```diff
@@ -71,17 +74,24 @@ $content-hub-two-col-breakpoint: $content-hub-width-three-module + px($gutter-wi
 
   @include respond-to($content-hub-one-col-breakpoint) {
     @include clearfix();
-    min-height: $content-hub-width-one-module;
+    min-height: $content-hub-width-one-module*1.1;
     padding-top: $baseline-unit*3;
   }
+
+  @include respond-to($content-hub-two-col-breakpoint) {
+    @include clearfix();
+    min-height: $content-hub-width-one-module*1.7;
+  }
 }
 
 .l-content-hub__hero-content {
   padding: 0 $content-hub-horizontal-space;
+  position: absolute;
+  top: 50%;
```

Comment:
If the image is treated as above then this need not be absolutely positioned and these two lines can be removed.